internal/flag/flags.go

```diff
@@ -15,241 +15,252 @@
+	AWSSecretKey                             = "awsSecretKey"                    // AWSSecretKey fag
+	S3AuthMethod                             = "s3AuthMethod"                    // S3AuthMethod flag
+	S3BucketEndpoint                         = "s3BucketEndpoint"                // S3BucketEndpoint flag
+	S3BucketName                             = "s3BucketName"                    // S3BucketName flag
+	S3MaxConnections                         = "s3MaxConnections"                // S3MaxConnections flag
+	DisableProxyS3                           = "disableProxyS3"                  // DisableProxyS3 flag
+	AcceptedTos                              = "acceptedTos"                     // AcceptedTos flag
+	SSEEnabled                               = "sseEnabled"                      // SSEEnabled flag
+	PathStyleAccessEnabled                   = "pathStyleAccessEnabled"          // PathStyleAccessEnabled flag
+	ReferenceTimeZoneOffset                  = "referenceTimeZoneOffset"         // ReferenceTimeZoneOffset flag
+	DailySnapshotRetentionDays               = "dailySnapshotRetentionDays"      // DailySnapshotRetentionDays flag
+	ClusterCheckpointIntervalMin             = "clusterCheckpointIntervalMin"    // ClusterCheckpointIntervalMin flag
+	SnapshotIntervalHours                    = "snapshotIntervalHours"           // SnapshotIntervalHours flag
+	SnapshotRetentionDays                    = "snapshotRetentionDays"           // SnapshotRetentionDays flag
+	WeeklySnapshotRetentionWeeks             = "weeklySnapshotRetentionWeeks"    // WeeklySnapshotRetentionWeeks flag
+	PointInTimeWindowHours                   = "pointInTimeWindowHours"          // PointInTimeWindowHours flag
+	ReferenceHourOfDay                       = "referenceHourOfDay"              // ReferenceHourOfDay flag
+	ReferenceMinuteOfHour                    = "referenceMinuteOfHour"           // ReferenceMinuteOfHour flag
+	MonthlySnapshotRetentionMonths           = "monthlySnapshotRetentionMonths"  // MonthlySnapshotRetentionMonths flag
+	Policy                                   = "policy"                          // Policy flag
+	SystemID                                 = "systemId"                        // SystemID flag
+	Default                                  = "default"                         // Default flag
+	Timestamp                                = "timestamp"                       // Timestamp flag
+	LiveMigrationDestinationClusterName      = "destinationClusterName"          // LiveMigrationDestinationClusterName flag
+	LiveMigrationDestinationProjectID        = "destinationProjectID"            // LiveMigrationDestinationProjectID flag
+	LiveMigrationHost                        = "migrationHost"                   // LiveMigrationHost flag
+	LiveMigrationSourceClusterName           = "sourceClusterName"               // LiveMigrationSourceClusterName flag
+	LiveMigrationSourceProjectID             = "sourceProjectID"                 // LiveMigrationSourceProjectID flag
```

Comment:
```suggestion
	LiveMigrationSourceProjectID             = "sourceProjectId"                 // LiveMigrationSourceProjectID flag
```

this file was hidden and just spotted it because you cemented it, for flags we stick to camelCase, this is to make our usage as close to shell as possible which prefers camelCase